dev.yml

```diff
@@ -7,7 +7,7 @@ up:
     - openssl
     - sqlite
   - ruby:
-      version: 2.2.3p172-shopify
+      version: 2.2.3
```

Comment:
https://github.com/Shopify/dev/pull/729
https://github.com/Shopify/dev-shopify/pull/96

Fixes this